Source/WebKit/WebProcess/Media/RemoteMediaSessionState.h

```diff
@@ -30,24 +30,28 @@
 #include <WebCore/MediaSessionGroupIdentifier.h>
 #include <WebCore/MediaSessionIdentifier.h>
 #include <WebCore/NowPlayingInfo.h>
+#include <WebCore/PageIdentifier.h>
 #include <WebCore/PlatformMediaSessionTypes.h>
 #include <wtf/LoggerHelper.h>
 #include <wtf/MediaTime.h>
 
-namespace WebCore {
-class PlatformMediaSessionInterface;
-}
-
 namespace WebKit {
 
 struct RemoteMediaSessionState {
+    WTF_DEPRECATED_MAKE_STRUCT_FAST_ALLOCATED(RemoteMediaSessionState);
```

Comment:
Do we need this considering it's deprecated ?

Reply:
Good point, switched to `WTF_MAKE_STRUCT_TZONE_ALLOCATED`